README.md

```diff
@@ -22,6 +22,16 @@ information using a variety of contributed modules and custom solutions.
 
 ## Install Instructions
 
+There are two ways to install Guardr.
+
+Option 1 - Install Fully Using Composer
+
+```
+composer create-project guardrdistro/guardr-project:8.x-dev some-dir --no-interaction
+```
+
+Option 2 Manual Install
```

Comment:
I would add a hyphen after `Option 2` to match how it is on line 27.

Reply:
The more I think about it, I would just remove the manual install and just make the composer create-project method the only install process. I am figuing if they have composer, that is the way to do it.

Reply:
Added a new commit to remove option 2